items.json

```diff
@@ -22205,8 +22205,8 @@
         "missileSpeed": 0,
         "stackAmount": 0,
         "length": 131,
-        "balance": 0.6,
-        "handling": 83,
+        "balance": 0.57,
+        "handling": 82,
         "bodyArmor": 0,
         "flags": [

```

Comment:
Not usable with onehand , yet has a onehanded mode

Reply:
2h flag issue